### ISSUE TITLE
make links open in new windows

### ISSUE THREAD

**Issue by melodykramer** [18F/hub#161]
particularly on the new hire page / how to page, it's easier to follow directions if the links open in a new window.

**Comment by meiqimichelle** [comment]
I suggest we leave this open as a broader problem of how-to content navigation. The best solution may be to restructure the how-to/new-hire somehow rather than opening in a new window. I don't think we fully solved the 'making info searchable/findable easily' yet :-/

**Comment by melodykramer** [comment]
You know, I was pointed to this today: 

http://18f.github.io/guides/

And the guides for Accessibility and Agile are really easy to navigate. I wonder if there's an already-existing template that we have internally that could be transferred...

**Comment by mbland** [comment]
@melodykramer: Yeah, @meiqimichelle opened #113 regarding the 18F Guides, and @elainekamlley and @blacktm are working with @kategarklavs to establish a team-wide template/model for these materials.

**Comment by meiqimichelle** [comment]
The thing we need to resolve, I think, is the searching-versus-browsing nav on the Hub. When we implemented search, the only way it would work in its current version is to have every 'answer' on its own page...and hence the current layout. If/when we resolve this, there are perhaps many better layouts.

@afeld Perhaps we need to see about freeing a day or two on our schedules to knock out search improvements (if that's possible for you given your workload).

**Comment by afeld** [comment]
> we need to see about freeing a day or two on our schedules to knock out search improvements

Fixit Day? :hammer: :muscle: :construction_worker:

**Comment by meiqimichelle** [comment]
True -- although I was hoping to focus on content for Fixit, but....perhaps :)
